docs/saml-2-0.md

```diff
@@ -8,16 +8,17 @@ To enabled SAML 2.0 support, define the following options in the configuration f
 ```json
 "sso": {
 	"saml": {
-		"required": true | false
+		"required": true | false,
+		"metadata_file": "config/sso/saml-idp-metadata.xml"
```

Comment:
For back-compat, need to leave it like this for now I think, unless you want me to add another default fallback in for this?

Reply:
Well if this is for v2 it doesn't necessarily need to be back compatible, and I don't think it's actually in use yet. But, we can still check the old location and default to `.config` because it seems pretty weird to have both a `config` and `.config` directories. That's just gonna cause confusion.